components/k8s-model-server/http-proxy/README.md

```diff
@@ -43,5 +43,47 @@ or
 
 - **Input and Output Schema**:
 
-In the example above, that correspones to a [tensorflow savedModel](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md) with input signature `"tf_model_input"` and `"image"` and output signature `"tf_model_output"`.
+In the example above, that correspones to a [tensorflow savedModel](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/python/saved_model/README.md) with method signature `tf.saved_model.signature_constants.PREDICT_METHOD_NAME`, input signature `"tf_model_input"` and `"image"` and output signature `"tf_model_output"`.
```

Comment:
Can you expand the text? It

Where is the signature def for the PREDICT_METHOD_NAME
Is it here?
https://github.com/tensorflow/serving/blob/master/tensorflow_serving/g3doc/signature_defs.md#classification-signaturedef

What do you mean by input signature "tf_model_input" and "image"

Reply:
> Where is the signature def for the PREDICT_METHOD_NAME
that's a constant defined in tensorflow `tensorflow.saved_model.signature_constants.PREDICT_METHOD_NAME`


input params with signature/key "tf_model_input" and "image"